attributes/config.rb

```diff
@@ -3,8 +3,12 @@
 # Config defaults options
 default['stackstorm']['config'] = {
   api_url: 'http://localhost:9101',
-  conf_root: '/etc',
+  api_host: '0.0.0.0',
```

Comment:
Shouldn't we bind to 127.0.0.1? Binding to all interfaces seems a little dangerous.